Modules/FIT/FV0/src/GenericCheck.cxx

```diff
@@ -82,6 +82,47 @@ void GenericCheck::configure()
   mCheckMinGraphLastPoint = getCheckFromConfig("MinGraphLastPoint");
   mCheckMaxGraphLastPoint = getCheckFromConfig("MaxGraphLastPoint");
 
+  // Set path to ccdb to get DeadChannelMap
+  if (auto param = mCustomParameters.find("ccdbUrl"); param != mCustomParameters.end()) {
+    setCcdbUrl(param->second);
+    ILOG(Debug, Support) << "configure() : using deadChannelMap from CCDB, configured url = " << param->second << ENDM;
+  } else {
+    setCcdbUrl("alice-ccdb.cern.ch");
+    ILOG(Debug, Support) << "configure() : using deadChannelMap from CCDB, default url = "
+                         << "alice-ccdb.cern.ch" << ENDM;
+  }
+
+  // Set internal path to DeadChannelMap
+  if (auto param = mCustomParameters.find("pathDeadChannelMap"); param != mCustomParameters.end()) {
+    mPathDeadChannelMap = param->second;
+    ILOG(Debug, Support) << "configure() : using pathDeadChannelMap: " << mPathDeadChannelMap << ENDM;
+  } else {
+    mPathDeadChannelMap = "FV0/Calib/DeadChannelMap";
+    ILOG(Debug, Support) << "configure() : using default pathDeadChannelMap: " << mPathDeadChannelMap << ENDM;
+  }
+
+  // Align mDeadChannelMap with downloaded one
+  std::map<std::string, std::string> metadata;
+  mDeadChannelMap = retrieveConditionAny<o2::fit::DeadChannelMap>(mPathDeadChannelMap, metadata, (long)-1);
+  if (!mDeadChannelMap->map.size()) {
+    ILOG(Error, Support) << "object \"" << mPathDeadChannelMap << "\" NOT retrieved (or empty). All channels assumed to be alive!" << ENDM;
+    mDeadChannelMap = new o2::fit::DeadChannelMap();
+    for (uint8_t chId = 0; chId < sNCHANNELS; ++chId) {
+      mDeadChannelMap->setChannelAlive(chId, 1);
+    }
+  }
+
+  // Print DeadChannelMap
+  mDeadChannelMapStr = "";
+  for (unsigned chId = 0; chId < mDeadChannelMap->map.size(); chId++) {
+    if (!mDeadChannelMap->isChannelAlive(chId)) {
+      mDeadChannelMapStr += (mDeadChannelMapStr.empty() ? "" : ",") + std::to_string(chId);
+    }
+  }
+  if (mDeadChannelMapStr.empty())
```

Comment:
The coding guidelines state that curvy brackets must be used even if there is a single line in the body of the `if` or `for`.

Reply:
Done